test/core/enqueue_links/enqueue_links.test.ts

```diff
@@ -942,5 +942,32 @@ describe('enqueueLinks()', () => {
                 expect(enqueued[i].options.forefront).toBe(true);
             }
         });
+
+        test('accepts skipNavigation option', async () => {
+            const enqueued: { request: Source; options: RequestQueueOperationOptions }[] = [];
+            const requestQueue = new RequestQueue({ id: 'xxx', client: apifyClient });
+
+            requestQueue.addRequests = async (requests, options) => {
+                // copy the requests to the enqueued list, along with options that were passed to addRequests,
+                // so that it doesn't matter how many calls were made
+                enqueued.push(...requests.map((request) => ({ request, options })));
+                return { processedRequests: [], unprocessedRequests: [] };
+            };
+
+            await cheerioCrawlerEnqueueLinks({
```

Comment:
your changes were related only to `enqueueLinksByClickingElements` method, and only to the playwright and puppeteer crawlers, why are you testing `enqueueLinks` with cheerio?

Reply:
hmm its actually missing in the base method too, so you will need to do the same here as well:

https://github.com/apify/crawlee/blob/master/packages/core/src/enqueue_links/enqueue_links.ts#L19

the `EnqueueLinksByClickingElementsOptions` should actually extend the base one so we remove the duplication (but that does not need to happen in this PR)

Reply:
Yeah, I fixed it and added a test to make sure it is working as expected.